html/css/signup.css

```diff
@@ -101,4 +101,281 @@ body {
         margin: 30px 20px;
         padding: 40px 15px 15px 15px;
     }
-}
\ No newline at end of file
+}
+  @import url("https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400;600&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto&display=swap");
+@import url("https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css");
+@import url("https://fonts.googleapis.com/css?family=Raleway:400,800");
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500&display=swap');
```

Comment:
If you write these 6 lines of codes on the first line of CSS file, that would be great.